### DISCUSSION TITLE
Hugo 0.95 - Update Notes

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#98] [Comments]
# Hugo 0.95 - Update Notes

This is the online playground of Patrick Kollitsch, David's Neighbour, in Germany produced, digital native living & working on Ko Samui, Thailand.

https://kollitsch.de/blog/2022/hugo-0-95-update-notes/

**Comment by sureshkumargondi** [comment]
Thanks for the summary. Super helpful. 

Hugo pushing ton of updates and hard to catchup.
